packages/headless/src/api/platform-client.test.ts

```diff
@@ -28,11 +29,47 @@ describe('url helper', () => {
 
   it('return the correct #analyticsUrl()', () => {
     allValidPlatformCombination().forEach((expectation) => {
-      expect(analyticsUrl(expectation)).toEqual(
-        expectation.analytics.split('/rest')[0]
-      );
+      if (!expectation.multiRegionSubDomain) {
+        expect(analyticsUrl(expectation)).toEqual(
+          expectation.analytics.split('/rest')[0]
+        );
+      }
     });
   });
+
+  it('return the correct #getOrganizationEndpoints()', () => {
```

Comment:
nitpik, this is a good candidate for an `it.each`
e.g. https://github.com/coveo/cli/blob/cf0b98146c5603220e81245c4fbea6c6d8fb9f66/packages/ui/atomic/health-check/src/index.spec.ts#L70-L79